pem/lib/pem/options.rb

```diff
@@ -15,6 +15,11 @@ def self.available_options
                                      description: "Renew the development push certificate instead of the production one",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :website_push,
```

Comment:
this new param conflicts with `development`, right? I think there is a way to note that.